### ISSUE TITLE
Cursor position messed up on reload when using antialiased font.

### ISSUE THREAD

**Issue by coopermaruyama** [atom/atom#6698]
To reproduce:

  1. Put in css: `-webkit-font-smoothing: subpixel-antialiased !important;`
  2. Open a file
  3. reload editor with `ctrl+alt+cmd+L` or close and open the same project.
  4. Should cause the below effect (note: I'm clicking at the end of line each time before typing)
:
    ![demo](https://s3.amazonaws.com/f.cl.ly/items/2J291S1C1K0C1D3a432D/Screen%20Recording%202015-05-07%20at%2008.33%20PM.gif)
   Also here's a more [high quality demo](http://cl.ly/0e390i1J3d0r). Notice how the cursor is not rendering at the end of the line where I'm clicking, although when I type, the text is correctly being inserted where it should. 

Side note: while investigating this I inspected the editor buffer's displayBuffer, and notice that it has a property called `charWidthsByScope`. I compared some of the `charWidths` before and after the reload and noticed that after reload they are all the same, whereas before the reload they had different values: ![screnshot](https://s3.amazonaws.com/f.cl.ly/items/2Q08350N3m2P1V022k26/Image%202015-05-07%20at%208.38.00%20PM.png) (top is after reload, bottom is before reload).

Also if you know how I can manually force a 'recalculation' of these values that would be great, as users of my syntax theme are having this issue and I'd like to fix it by forcing the recalculation of these values.

**Comment by benogle** [comment]
Is this still an issue?

**Comment by coopermaruyama** [comment]
Nope! A recent update seems to have fixed it finally!  

I haven't been keeping a close eye on the recent changes but in my package I originally got around this by manually running `editorView.component.linesComponent.remeasureCharacterWidths()` (or something like that, don't remember exactly) but today I tried removing the hack and it worked fine even with it removed, so that's great! It actually looks like that method was removed in [this commit](https://github.com/atom/atom/commit/72093da1a7a6513bacff388df4488a28627e9e77#diff-096d4c7e33085a5833e64a121f09e102) so I'd guess that it got fixed somewhere around that time.

Thanks!
